gtsam/linear/NoiseModel.h

```diff
@@ -684,9 +692,9 @@ namespace gtsam {
       { Vector b; Matrix B=A; this->WhitenSystem(B,b); return B; }
       inline virtual Vector unwhiten(const Vector& /*v*/) const
       { throw std::invalid_argument("unwhiten is not currently supported for robust noise models."); }
+      // Fold the use of the m-estimator residual(...) function into distance(...)
       inline virtual double distance(const Vector& v) const
-      { return this->whiten(v).squaredNorm(); }
-      // TODO(mike): fold the use of the m-estimator residual(...) function into distance(...)
+      { return robust_->residual(this->unweightedWhiten(v).norm()); }
       inline virtual double distance_non_whitened(const Vector& v) const
       { return robust_->residual(v.norm()); }
       // TODO: these are really robust iterated re-weighting support functions
```

Comment:
Is this still a TODO? Seems more like a comment but I am not sure. If it isn't a TODO, don't worry about it, just asking since I can clean it up later.